.storybook/pages/CadStep1/CadStep1.module.css

```diff
@@ -0,0 +1,70 @@
+/*------------------------------------*\
+    # CAD Step 1
+\*------------------------------------*/
+
+/**
+ * Page Layout
+ */
+.cad-step-1__layout {
+  padding: var(--eds-size-2);
+  @media all and (min-width: $eds-bp-md) {
+    padding: var(--eds-size-10);
+  }
+}
+
+/**
+ * Page Layout Left Section
+ */
+.cad-step-1__left-section {
```

Comment:
I feel like we should either wrap this `Layout` in a `LayoutContainer` and align it with the content above or create a `variant="narrow"` version of this. Let's align with design on this.

Reply:
I agree with removing the page specific styling, and with it gone, the left/right section styling should also be gone. I will sync with designers on dividing the header to match the body.